test/database.js

```diff
@@ -1,17 +1,80 @@
 'use strict';
 
+/* eslint new-cap: 0 */
+const metatests = require('metatests');
 const metaschema = require('..');
+const { attribute: { Enum, Many, Include } } = require('../lib/decorators');
 
-metaschema.load('general', (err, schema) => {
-  if (err) throw err;
-  metaschema.build(schema);
+const domains = {
+  nomen: {
+    domain: 'Nomen',
+    definition: { type: 'string', length: 60, control: 'InputBox' }
+  },
+  nomenRequiredTrue: {
+    domain: 'Nomen',
+    required: true,
+    definition: { type: 'string', length: 60, control: 'InputBox' }
+  },
+  sex: {
+    domain: 'Sex',
+    definition: Enum('female', 'male')
+  },
+  born: {
+    domain: 'DateDay',
+    definition: { type: 'number', format: 'yyyy-mm-dd' }
+  }
+};
 
-  const Person = metaschema.categories.get('Person');
-  console.dir(Person, { depth: null });
+const categoriesConfigs = {};
 
-  const Language = metaschema.categories.get('Language');
-  console.dir(Language, { depth: null });
+categoriesConfigs.FullName = {
+  name: 'FullName',
+  definition: {
+    FirstName: domains.nomenRequiredTrue,
+    MiddleName: domains.nomen,
+    Patronymic: domains.nomen,
+    Surname: domains.nomenRequiredTrue
+  }
+};
 
-  const FullName = metaschema.categories.get('FullName');
-  console.dir(FullName, { depth: null });
+categoriesConfigs.Language = {
+  name: 'Language',
+  definition: {
+    Name: domains.nomenRequiredTrue
+  }
+};
+
+categoriesConfigs.Person = {
+  name: 'Person',
+  definition: {
+    FullName: Include({
+      category: 'FullName',
+      definition: categoriesConfigs.FullName.definition
+    }),
+    Sex: domains.sex,
+    Born: domains.born,
+    Languages: Many({
+      category: 'Language',
+      definition: categoriesConfigs.Language.definition
+    })
+  }
+};
+
+metatests.test('Database / database', (test) => {
+  metaschema.load('general', (err, schema) => {
+    test.assertNot(err);
+    metaschema.build(schema);
+
+    for (const categoryName in categoriesConfigs) {
+      const config = categoriesConfigs[categoryName];
+      const category = metaschema.categories.get(categoryName);
+
+      test.strictSame(config.name, category.name);
```

Comment:
Argument order is incorrect, its `actual, expected`. This should be `test.strictSame(category.name, config.name);`